examples/examples/pretty_print.rs

```diff
@@ -0,0 +1,11 @@
+use iceoryx2::service::{service_name::ServiceName, zero_copy, Service};
```

Comment:
I think this is a feature that does not require a separate example.

Reply:
Sounds reasonable.